data/hai/hai ships.txt

```diff
@@ -11,6 +11,88 @@
 # You should have received a copy of the GNU General Public License along with
 # this program. If not, see <https://www.gnu.org/licenses/>.
 
+ship "Anomalocaris"
+	plural "Anomalocarises"
+	sprite "ship/hai anomalocaris"
+	thumbnail "thumbnail/hai anomalocaris"
+	attributes
+		category "Medium Warship"
+		"cost" 7400000
+		"shields" 14200
+		"hull" 6800
+		"required crew" 20
+		"bunks" 26
+		"mass" 970
+		"drag" 9.3
+		"heat dissipation" .49
+		"fuel capacity" 400
+		"cargo space" 121
+		"outfit space" 533
+		"weapon capacity" 192
+		"engine capacity" 98
+		weapon
+			"blast radius" 210
+			"shield damage" 2100
+			"hull damage" 1050
+			"hit force" 3150
+	outfits
+		`"Biroo" Atomic Thruster`
+		`"Bondir" Atomic Steering`
+		"Boulder Reactor"
+		"Chameleon Anti-Missile" 2
+		"Hai Diamond Regenerator"
+		"Hai Ravine Batteries"
+		"Hai Williwaw Cooling" 3
+		Hyperdrive
+		"Ion Cannon"
+		"Pebble Core"
+		"Pulse Cannon"
+		"Pulse Rifle" 20
+		"Pulse Turret" 2
+		"Quantum Keystone"
+	gun -24 -89.5
+	gun 24 -89.5
+	turret -40 -28
+	turret 40 -28
+	turret 0 60
+	turret 0 -32.5
+	engine -50 15
+	engine 50 15
+	"steering engine" -50 15
+		zoom 0.5
+		right
+	"steering engine" 50.5 15
+		zoom 0.5
+		left
+	leak "leak" 40 50
```

Comment:
```suggestion
	engine 50 15
	leak "leak" 40 50
```
The steering engines being the same engines, in this particular case, don't really look that good with the ship's mass center especially when turning without forward thrust. It looks less awkward to not have them here, or add small tiny ones to more appropriate places later.